package.json

```diff
@@ -5,6 +5,9 @@
   "main": "src/app.js",
   "dependencies": {
     "babel-loader": "^8.0.5",
+    "normalize-scss": "^7.0.1",
```

Comment:
As you're having it as dep you can get rid of the css version right? and import it into your main scss file. It make sense?